### PR TITLE
Use range-v3 loops(transform)

### PR DESCRIPTION
**Pull request by anurag-git**
Partial fix for #10738 
Replaced boost::adaptors::transformed() to ranges::views::transform

### CODE REVIEW

File: scripts/install_deps.sh [hunk 1]
```diff
@@ -93,8 +93,8 @@ case $(uname -s) in
             10.15)
                 echo "Installing solidity dependencies on macOS 10.15 Catalina."
                 ;;
-            11.0 | 11.1)
-                echo "Installing solidity dependencies on macOS 11.0 / 11.1 Big Sur."
+            11.0 | 11.1 | 11.2)
+                echo "Installing solidity dependencies on macOS 11.0 / 11.1 Big Sur/ 11.2 Big Sur."
```
Comment (chriseth) [inline scripts/install_deps.sh]: Can you put this into its own PR please?

**Comment by chriseth** [comment]
Please use rebase instead of merge.

**Comment by anurag-git** [comment]
> Please use rebase instead of merge.

Tried, lets me see

**Comment by chriseth** [comment]
Please note that `remove` and `remove_erase` are two different things. Can you remove (haha) these changes from the PR, than we can merge all the transform changes first?

**Review by chriseth — APPROVED** [review]

File: scripts/install_deps.sh (hunk 1 above)
Comment (anurag-git) [reply scripts/install_deps.sh]: Done in PR #11231
